CaptchaBot/Services/WelcomeService.cs

```diff
@@ -169,16 +197,11 @@ public async Task ProcessNewChatMember(Message message)
 
         private static string GetPrettyName(User messageNewChatMember)
         {
-            var names = new List<string>(3);
-
-            if (!string.IsNullOrWhiteSpace(messageNewChatMember.FirstName))
-                names.Add(messageNewChatMember.FirstName);
-            if (!string.IsNullOrWhiteSpace(messageNewChatMember.LastName))
-                names.Add(messageNewChatMember.LastName);
-            if (!string.IsNullOrWhiteSpace(messageNewChatMember.Username))
-                names.Add("(@" + messageNewChatMember.Username + ")");
-
-            return string.Join(" ", names);
+            var array = new string[] { messageNewChatMember.FirstName ,
+                                       messageNewChatMember.LastName,
+                                       messageNewChatMember.Username 
+            };
+            return string.Join(" ", array.Where(q => !string.IsNullOrWhiteSpace(q)));
```

Comment:
Here, the code semantic was changed. Earlier, it was adding `"(@" + messageNewChatMember.Username + ")"`, but now it's just a `messageNewChatMember.Username`.